src/h2load.cc

```diff
@@ -2539,10 +2687,17 @@ int main(int argc, char **argv) {
   double rps = 0;
   int64_t bps = 0;
   if (duration.count() > 0) {
-    auto secd = std::chrono::duration_cast<
-        std::chrono::duration<double, std::chrono::seconds::period>>(duration);
-    rps = stats.req_success / secd.count();
-    bps = stats.bytes_total / secd.count();
+    if (config.warm_up_time > 0) {
+      // we only want to consider the rate-period if warm-up is given
+      rps = stats.req_success / config.rate_period;
```

Comment:
It's unclear to me why rate period is in the denominator here. Isn't the idea that we divide by the total amount of time not including warmup or tear down? The period is the applied request rate rather, or did I read it wrong?